Ejercicios/02 - Página Básica en ES6/src/index.html

```diff
@@ -53,8 +53,10 @@ <h1>Este es un ejemplo</h1>
         commodo. Donec id porta nisi. Suspendisse eleifend urna felis, eu rhoncus nibh pulvinar in. In ut nibh viverra,
         maximus turpis a, tempus metus.
       </p>
-
+      <div id="botonpausa"><input type="button" value="Pausa/Inicio"></div>
```

Comment:
```suggestion
      <input id="botonpausa" type="button" value="Pausa/Inicio">
```
En JavaScript necesitamos el elemento `<input>` y no el `<div>` por eso el id debe asignarse al input y en ese sentido sobra el `<div>`.
Hay que hacer lo mismo con el otro botón.